### ISSUE TITLE
Site timezone defaults to UTC instead of user timezone

### ISSUE THREAD

**Issue by startuptester** [wordpress-mobile/WordPress-iOS#16231]
From user feedback session:

This comes up very often in testing so it's obvious it confuses people. 

You schedule a post for later and it publishes immediately but you have no idea why. It's because new site timezones default to UTC instead of the user's location. This is not an intuitive experience.

**Comment by startuptester** [comment]
Android twin: https://github.com/wordpress-mobile/WordPress-Android/issues/14400

**Comment by guarani** [comment]
I tested this on WordPress for iOS and as mentioned above, the cause of the issue is related to how WP.com sites created via the app default to a timezone of UTC-00:00.
However, there is [another issue that may prevent scheduled posts from publishing](https://github.com/wordpress-mobile/WordPress-iOS/issues/10456): only after a site receives visits (page view stats) do scheduled posts "work", before that, they remain stuck in the scheduling queue. I could reproduce this secondary issue in tests today.

What follows are screenshots showing how sites created on mobile use the (incorrect) UTC timezone, while sites created on WP.com use the correct (local) timezone. 

| WPiOS - ❌ Incorrect timezone (defaults to UTC) | Web - ✅ Correct timezone (defaults to local timezone) |
| - | - |
| <img src="https://user-images.githubusercontent.com/1898325/151037339-054c81fe-f149-4284-b32d-2375c5d0be60.jpeg" width="400"> | <img width="800" alt="Screen Shot 2022-01-25 at 15 25 26" src="https://user-images.githubusercontent.com/1898325/151037896-70da87ec-716f-4074-8fd0-7ed46d83fedc.png"> |

The next step here is to check whether WordPress for iOS is failing to send a timezone parameter (or similar) to the backend when creating a site.

**Comment by guarani** [comment]
Here are the requests made on each platform when a site is created:

| | Web | WordPress for iOS | Comment
| - | - | - | - |
| Request URL | https://public-api.wordpress.com/rest/v1.1/sites/new?http_envelope=1 | https://public-api.wordpress.com/rest/v1.1/sites/new?locale=en | Practically same ✅ |
| Request payload | <pre><br>{<br>&nbsp;&nbsp;"client_id": "123456",<br>&nbsp;&nbsp;"client_secret": "CLIENT SECRET HERE",<br>&nbsp;&nbsp;"find_available_url": true,<br>&nbsp;&nbsp;"public": 0,<br>&nbsp;&nbsp;"blog_name": "blognamehere",<br>&nbsp;&nbsp;"blog_title": "blogtitle",<br>&nbsp;&nbsp;"options": {<br>&nbsp;&nbsp;&nbsp;&nbsp;"site_information": {<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"title": "blogtitle"<br>&nbsp;&nbsp;&nbsp;&nbsp;},<br>&nbsp;&nbsp;&nbsp;&nbsp;"lang_id": 1,<br>&nbsp;&nbsp;&nbsp;&nbsp;"site_creation_flow": "gutenboarding",<br>&nbsp;&nbsp;&nbsp;&nbsp;"enable_fse": true,<br>&nbsp;&nbsp;&nbsp;&nbsp;"theme": "pub/blank-canvas-blocks",<br>&nbsp;&nbsp;&nbsp;&nbsp;"timezone_string": "America/Asuncion",<br>&nbsp;&nbsp;&nbsp;&nbsp;"template": "blank-canvas-blocks",<br>&nbsp;&nbsp;&nbsp;&nbsp;"use_patterns": true,<br>&nbsp;&nbsp;&nbsp;&nbsp;"selected_features": [],<br>&nbsp;&nbsp;&nbsp;&nbsp;"wpcom_public_coming_soon": 1,<br>&nbsp;&nbsp;&nbsp;&nbsp;"is_blank_canvas": true<br>&nbsp;&nbsp;},<br>&nbsp;&nbsp;"validate": false<br>}</pre> | <pre>{<br>&nbsp;"client_secret": "CLIENT SECRET HERE",<br>&nbsp;"public": 1,<br>&nbsp;"options": {<br>&nbsp;&nbsp;"template": "about",<br>&nbsp;&nbsp;"site_information": {<br>&nbsp;&nbsp;&nbsp;"site_tagline": ""<br>&nbsp;&nbsp;}<br>&nbsp;},<br>&nbsp;"blog_name": "blognamehere",<br>&nbsp;"client_id": "11",<br>&nbsp;"validate": true,<br>&nbsp;"lang_id": "1",<br>&nbsp;"blog_title": "Site Title"<br>}</pre> | ⚠️ WPiOS is missing a <code>timezone_string</code> in the options object |

The next step would be to try adding the `timezone_string` to the site creation request and seeing if that allows WPiOS to create sites using the user's timezone.
